CMakeModules/installer.cmake

```diff
@@ -0,0 +1,37 @@
+include(InstallRequiredSystemLibraries)
+
+set(CPACK_PACKAGE_CONTACT "someone@eos.org")
+set(CPACK_OUTPUT_FILE_PREFIX ${CMAKE_BINARY_DIR}/packages)
+set(CMAKE_INSTALL_PREFIX ${CMAKE_BINARY_DIR}/install)
+
+SET(CPACK_PACKAGE_DIRECTORY "${CMAKE_INSTALL_PREFIX}")
+set(CPACK_PACKAGE_NAME "eos")
+set(CPACK_PACKAGE_VENDOR "EOSIO")
+set(CPACK_PACKAGE_VERSION_MAJOR "${VERSION_MAJOR}")
+set(CPACK_PACKAGE_VERSION_MINOR "${VERSION_MINOR}")
+set(CPACK_PACKAGE_VERSION_PATCH "${VERSION_PATCH}")
+set(CPACK_PACKAGE_VERSION "${CPACK_PACKAGE_VERSION_MAJOR}.${CPACK_PACKAGE_VERSION_MINOR}.${CPACK_PACKAGE_VERSION_PATCH}")
+set(CPACK_PACKAGE_DESCRIPTION "A client for the Eos network")
```

Comment:
We've had an official update on the name and this should be "EOS.IO"

Reply:
Package vendor or name or in description?

Reply:
name and description.  Vendor can be "block.one".  I think @jgiszczak  is determining the proper package contact address.

Reply:
The package contact address should be "support@block.one".

Reply:
done